routes/flock.json

```diff
@@ -119,43 +119,282 @@
+          "For Competition V, we used industry-leading Hyperloop-spec brakes produced by W.C. Branham in the USA. The brakes use coiled springs to apply the friction pads to the rail, and they are bolted to the frame with a support plate to ensure a robust mechanical system. The brakes are placed at the front and rear of the pod to maximize stopping force."
+        ],
+        "parallax": "/images/flock/goose5/brakes.png"
+      },
+      {
+        "title": "Shell",
+        "content": [
+          "The shell provides an external layer to the pod that decreases turbulence and stops foreign debris from entering the inner components. The shell design balances minimal manufacturing complexity and cost with aesthetical aspects for branding and promotional material."
+        ],
+        "parallax": "/images/flock/goose5/g5-shell.png"
+      },
+      {
+        "title": "Motor Controller",
+        "content": [
+          "The motor controller ultimately controls the Linear Induction Motors, and is split into 2 main portions: the power board, and the control board. The power board produces the 3-phase AC needed to power the LIM. This is achieved through the use of a IGBT 3-phase inverter. The control board acts as a medium between the power board and the microcontroller. Multiple systems are also used as sensors and protectors of the boards."
+        ],
+        "parallax": "/images/flock/goose5/motor-controller-schematics.png"
+      },
+      {
+        "title": "Battery and Battery Management",
+        "content": [
+          "The battery management system (BMS) is used to monitor and balance the cell array used to power the Linear Induction Motors and low voltage systems. The lithium ion cells experience multiple charge and discharge cycles, thus they will degrade if improperly managed without the BMS. The BMS also utilizes passive cell balancing to ensure that all batteries in the stack are evenly charged, thereby prolonging cell life and maintaining healthy charge levels."
+        ],
+        "parallax": "/images/flock/goose5/battery.png"
+      },
+      {
+        "title": "Embedded Sensors",
+        "content": [
+          "The pod consists of an array of 32 sensors, with 11 unique models. A few examples of models would be the Piezo, which measures vibrations both at the wheel and frame level, and the Phototransitor that reads the RPM of the pod's wheels for calculating speed."
+        ]
+      },
+      {
+        "title": "Physical System Architecture: Communications",
+        "content": [
+          "There are 2 major components comprising the communications system:",
+          "<ul><li>The Networking and TCP component utilizes a Desktop Raspberry Pi. It facilitates relaying sensor data from the sensors to the desktop dashboard to be displayed, and to send commands from the remote desktop dashboard back to the pod.</li>",
```

Comment:
facilitates relaying sensor data 

to 

it relays sensor data

Reply:
to send commands from the remote desktop dashboard back to the pod.

to 

sends commands from the remote desktop dashboard back to the pod